eslint/.eslintrc.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
paste file .eslintrc.json in main directory, near main package json and delete folder eslint